modules/data-entry/src/main/frontend/src/dataHomepage/ExportButton.jsx

```diff
@@ -443,22 +448,39 @@ function ExportButton(props) {
                 </Grid>
               )}
               <FormControl variant="standard" fullWidth className={classes.variableDropdown}>
-                <InputLabel id="label">Select questions/sections from this questionnaire</InputLabel>
-                <Select
-                  variant="standard"
-                  labelId="label"
-                  value={tempValue}
-                  label="Select questions/sections from this questionnaire"
-                  onChange={handleEntitySelected}
-                >
-                  { entities?.filter(v => !selectedEntityIds.includes(v.path))
-                      .map(v =>
-                        <MenuItem value={v.path} key={`option-${v.name}`} className={classes.variableOption}>
-                          { getAvatar(v.type) }
-                          <ListItemText primary={v.name} secondary={v.text} />
-                        </MenuItem>)
+                <Autocomplete
+                  open={autocompleteOpen}
+                  onOpen={() => {
+                    setAutocompleteOpen(true);
+                  }}
+                  onClose={() => {
+                    setAutocompleteOpen(false);
+                  }}
+                  value={tempValue && variables.find(item => item.path == tempValue) || null}
+                  PopperComponent={FilterPopper}
```

Comment:
We should use the built-in popper which is nicely displayed under the input as a proper dropdown of the same width.  That would make the use of the `open` state unnecessary as it's already handled internally.

```suggestion
                  value={tempValue && variables.find(item => item.path == tempValue) || null}
```

Reply:
Done this for ReferenceInput, see b460502 for inspiration.